base_import_async/models/base_import_import.py

```diff
@@ -55,10 +55,9 @@ def execute_import(self, fields, columns, options, dryrun=False):
             translated_model_name = search_result[0][1]
         else:
             translated_model_name = self._description
-        description = _("Import %(model)s from file %(from_file)s") % {
-            "model": translated_model_name,
-            "from_file": self.file_name,
-        }
+        description = _(
+            "Import %s from file %s", translated_model_name, self.file_name
+        )
```

Comment:
```suggestion
        description = _(
            "Import %(model)s from file %(from_file)s", model=translated_model_name, from_file=self.file_name
        )
```

It's important to 
1/ keep the source string unchanged
2/ use keyword placeholders so it can be translated to languages that need another order of the words